### PR TITLE
Query update animals

### PR DESCRIPTION
**Pull request by kkmanuu**
✔️ 
 I added data to the animals table. Inside a transaction update the animals table by setting the species column to unspecified
Screenshot (68)

✔️ 
I Rollback the species and verify the species columns
Screenshot (69)

✔️ 
I Update the animals table by setting the species column to digimon for all animals that have a name ending in mon. and Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Screenshot (70)

✔️ 
I Verify changes  and Commit the transaction.
Screenshot (71)

✔️ 
I delete all records in the animals table.
Screenshot (72)

✔️ 
I Rollback Transactions
Screenshot (73)

✔️ 
 I Delete all animals born after Jan 1st, 2022 and Create a savepoint for the transaction.
Screenshot (74)

✔️ 
I UPDATE animals
SET weight_kg = weight_kg * -1
WHERE weight_kg < 0; and SELECT COUNT(*) AS total_animals
FROM animals;
Screenshot (75)

✔️ 
I SELECT COUNT(*) AS non_escape_animals
FROM animals
WHERE escape_attempts = 0; and SELECT
AVG(weight_kg) AS average_weight
FROM
animals;
Screenshot (76)

✔️ 
I  SELECT neutered, SUM(escape_attempts) AS total_escapes
FROM animals
GROUP BY neutered
ORDER BY total_escapes DESC
LIMIT 1; and SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight
FROM animals
GROUP BY species;
Screenshot (77)

✔️ 
 I SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight
FROM animals
GROUP BY species; and SELECT species, AVG(escape_attempts) AS avg_escape_attempts
FROM animals
WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
GROUP BY species;
Screenshot (78)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -55,4 +55,81 @@ animals
...
+
+COMMIT;
+
+SELECT * FROM animals;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```
Comment (AbelG101) [inline queries.sql]: - [ ] Ouch! this is a close one but the above query has to be in a separate transaction so please make sure to add a `BEGIN` statement before deleting.

File: queries.sql [hunk 2]
```diff
@@ -55,4 +55,81 @@ animals
...
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```
Comment (AbelG101) [inline queries.sql]: - [ ] I noticed the screenshot for this query has a commit statement which shouldn't be part of the transaction and is creating a mismatch between your query and the corresponding results, so please take a screenshot with the proper query.
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @kkmanuu 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Species column is properly added ✔
- Proper transaction management ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a good screenshot summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkmanuu** [comment]
1 Remove commit and take another screenshot 
Screenshot (80)

File: queries.sql (hunk 2 above)
Comment (kkmanuu) [reply queries.sql]: @AbelG101   I'm done

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @kkmanuu , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes implemented 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**